tendrl/commons/jobs/__init__.py

```diff
@@ -96,6 +111,17 @@ def run(self):
     def stop(self):
         pass
 
+    def read_job(self, key):
+        result = {}
+        job = tendrl_ns.etcd_orm.client.read(key)
+        if hasattr(job, '_children'):
+            for item in job._children:
+                if "dir" in item:
+                    result[item["key"].split("/")[-1]] =  self.read_job(item["key"])
+                else:
+                    result[item["key"].split("/")[-1]] = item["value"]
```

Comment:
aah it is recursive function :+1:  recurring @ line 120 ok I didn't realize this before